webclient/webclient/src/main/java/io/helidon/webclient/NettyClientHandler.java

```diff
@@ -144,18 +146,23 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpObject msg) throws IO
             requestConfiguration.cookieManager().put(requestConfiguration.requestURI(),
                                                      clientResponse.headers().toMap());
 
+            // Set entity to channel if further forwarding is required
```

Comment:
You are not setting here :-) You are actually getting from the channel.

Reply:
Yep